src/components/sidebar-item/sidebar-item.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import StatusTag from '../status-tag/status-tag';
+import { type StatusTagType } from '../status-tag/status-tag';
+import Flex from '../../../src/helpers/components/Flex/Flex';
+import { ElementContent } from '../../types';
+
+const SidebarItem: React.FC<{ children: ElementContent; status: StatusTagType }> = ({ children, status }) => {
```

Comment:
Let's try to promote here same standards that we have in Vibe and declare these props as an `interface SidebarItemProps` 🙏